### DISCUSSION TITLE
How to make freeDrawingBrush.color repeat draw, opacity color does not overlap?

### DISCUSSION THREAD

**Discussion by Lynne42** [fabricjs/fabric.js#7177] [Q&A]
## Version
4.5.1

## Test Case
http://jsfiddle.net/k6uo2mcd/2/

## Information about environment
chrome  browser

**How to make freeDrawingBrush.color repeat draw, opacity color does not overlap?**

![image](https://user-images.githubusercontent.com/46595240/124258577-250ec380-db60-11eb-9057-bf2238cf4f2f.png)

**Hope:**

After repeated draws, the color should be the same (I need to use a transparent color to see the background image)

**Comment by asturur** [comment]
you can try using a different globalCompositeOperation for the brush.
I'm not 100% sure is still supported.
https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation

**Comment by voigtito** [comment]
Did you guys came up with a solution for this?

**Reply by webwyb** [reply]
Hi bro, I have the same problem, have you found a solution?
